pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -587,18 +587,11 @@ func TestStorageRequests(t *testing.T) {
 		name       string
 		wantStatus *framework.Status
 	}{
-		{
```

Comment:
Why remove this? It's a valuable one.

Reply:
The function name here is TestStorageRequests, but I don't think it storage related one.
TC name is "due to container scratch disk", but it returns CPU error.

Reply:
We have similar test cases in `TestEnoughRequests`, so sgtm.